src/plugins/Workers/TransactionSyncStreamWorker/TransactionSyncStreamWorker.spec.js

```diff
@@ -0,0 +1,556 @@
+const chai = require('chai');
```

Comment:
Is it a unit test now or still integration? If still integration then we need to move it to  `test/integration/plugins/Workers/TransactionSyncStreamWorker/TransactionSyncStreamWorker.spec.js`

Reply:
Done